_summaries/kang2019decoupling_2.md

```diff
@@ -1,19 +1,86 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: kang2019decoupling
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: joshpapermaster
+score: 7 # How did you like this paper 0(dislike) to 10(love)
 ---
 
-TODO: Summarize the paper:
+<!-- TODO: Summarize the paper:
 * What is the core idea?
 * How is it realized (technically)?
 * How well does the paper perform?
-* What interesting variants are explored?
+* What interesting variants are explored? -->
+## What is the core idea?
+Visual recognition solutions can struggle when there is a long-tailed distribution, which is also known as the class imbalance problem. This paper separates the this problem into two processes: representation learning and classification. 
+
+Representation learning
+- trained with different sampling techniques and losses
+
+Classification
+- the long tailed classes are recognized by the model from various classifiers  
+
+The paper determined the following findings from using their decoupled process:
+- instance based sampling provided really good generalized results
+- during representation learning it's beneficial to adjust decision boundaries of classifier 
+    - done by sampling or weight normalization
+- Can apply to any standard network 
+
+## How is it realized technically?
+
+Experiments covered the following different decoupled techniques to determine which were most effective
+
+Representation Learning:
+- Instance-balanced sampling
+    - All examples have same chance of being sampled
+- Class-balanced sampling
+    - Each class has same chance of being sampled and then example uniformly sampled within the class
+- Square-root sampling
+    - Middle ground between the two previous sampling techniques. Square-roots the total number of images in each class when calculating the ratio of images in specific class / total images
+- Progressively-balanced sampling
+    - Uses instance-balanced sampling for some epochs and then switches to class-balanced sampling
```

Comment:
I think they didn't do the hard-assignment version, they do the softer version where they interpolate these two kinds of sampling